packages/admin/admin-generator/src/commands/generate/config/transformConfig.ts

```diff
@@ -179,22 +198,28 @@ function getTypePropertyFromObjectLiteral(node: ts.ObjectLiteralExpression) {
 }
 
 // visits ast and collects all imports statements in the source file
-function collectImports(rootNode: ts.Node) {
-    const importedIdentifiers = new Map<string, { name: string; import: string }>();
+export function collectImports(rootNode: ts.Node) {
+    const importedIdentifiers = new Map<string, { name: string; import: string; defaultImport?: boolean }>();
```

Comment:
Here default imports are not supported as well (needed it for p-debounce)

(the export was added because this function is now tested)